.github/workflows/build-and-test.yml

```
@@ -34,8 +34,28 @@ jobs:
       - name: "Run flox version"
         run: $FLOXBIN --version
 
-      - name: "Enable build features"
-        run: $FLOXBIN config --set-bool features.build true
-
       - name: "Run make"
         run: make
+
+  report-failure:
+    name: "Report Failure"
+    runs-on: "ubuntu-latest"
+    timeout-minutes: 30
+
+    if: ${{ failure() && github.event_name == 'workflow_dispatch' }}
```

(non-blocking) This would prevent false-positives if people trigger workflows on feature branches and will get us reporting when we run the workflow on `main` in addition to PRs.
```suggestion
    if: ${{ failure() && (github.base_ref == 'refs/heads/main' || github.ref == 'refs/heads/main') }}
```